src/integrationTest/resources/application.yml

```diff
@@ -48,3 +48,10 @@ os:
     port: 8126
     url: ${os.api.hostname}:${os.api.port}
     key: key
+
+av:
+  enabled: false
+  api:
+    hostname: localhost
+    port: 8128
+    timeout: 20000
```

Comment:
Are these required if av isn't enabled?

Reply:
Probably not - but the ClamAVClient is constructed whether or not AV is enabled.
It's a runtime call to the AvScanService.scan() method that checks the enabled flag before calling the external service.

Reply:
The integration tests fail if either the api section is missing, or the hostname etc are blank.